src/ar100/Makefile

```diff
@@ -0,0 +1,41 @@
+CROSS_PREFIX=or1k-linux-musl-
+dirs-y += src/generic src/ar100 lib/ar100
+
+CFLAGS		+= -O3
+CFLAGS		+= -fno-builtin
+CFLAGS		+= -fno-pie
+CFLAGS		+= -ffreestanding
+CFLAGS		+= -msfimm -mshftimm -msoft-div -msoft-mul
+CFLAGS		+= -Ilib/ar100
+CFLAGS_klipper.elf := $(CFLAGS) -Wl,-T,src/ar100/ar100.ld -Wl,-Map,out/klipper.map
+CFLAGS_klipper.elf += -Wl,--no-undefined  -Wl,--gc-sections -static
+CFLAGS_klipper.elf += -Wl,--no-dynamic-linker -Wl,--allow-multiple-definition
+
+SFLAGS		 = -nostdinc -Wa,--fatal-warnings -MMD -Wmissing-prototypes -Wall -Wextra
```

Comment:
The extra warning flags (`-W...`) end up causing unnecessary code churn - users end up running a variety of different versions of gcc which often don't agree on what a valid warning is.

Reply:
I've cleaned up the Makefile a little.